src/components/navigation/EdgeLogoHeader.tsx

```diff
@@ -1,14 +1,16 @@
 import * as React from 'react'
-import { Image, View } from 'react-native'
+import { View } from 'react-native'
+import FastImage from 'react-native-fast-image'
 
+import edgeMark from '../../assets/images/edgeLogo/Edge_logo_Icon.png'
 import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 
 export function EdgeLogoHeader() {
   const theme = useTheme()
   const styles = getStyles(theme)
   return (
     <View style={styles.container}>
-      <Image style={styles.icon} source={theme.headerIcon} resizeMode="contain" />
```

Comment:
The reason the header image comes from the theme is because we have white-label versions of our app with different logos. Simply swapping out the theme allows us to re-brand the app as needed. So, please put the source back the way it was.

Reply:
Reverted, and now uses a new `ImageProp` declared in `Theme.ts`.